### PR TITLE
DOC-9648: Separate the definitions of boolean and boolean field queries

### PR DESCRIPTION
**Pull request by abhinavdangeti**
+ Also fixing the example descriptions and formatting for-
    - boolean query
    - boolean field query

### CODE REVIEW

**Comment by abhinavdangeti** [comment]
@RayOffiah This one's ready for review.

Also, separately - would you give us the right set of permissions to add reviewers for this PRs?

File: modules/fts/pages/fts-supported-queries-boolean-query.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+= Boolean Query
+
+A boolean query is a compound query that allows for the user to employ a combination of conjunction and disjunction queries.
+It takes three groups of queries:
+
+* *must*: Result documents must satisfy all of these queries
+
+* *must_not*: Result documents should satisfy any of these queries
+
+* *should*: Result documents must not satisfy any of these queries
```
Comment (dnault) [inline modules/fts/pages/fts-supported-queries-boolean-query.adoc]: Are the descriptions of "should" and "must not" swapped?

Separate note: a disjunction query's "min" property works for both "should" and "must not", which lets a user express things like "MUST NOT satisfy [min] or more of these disjuncts", or "SHOULD satisfy at least [min] of these disjuncts". I don't know if it's worth documenting that here, but I found it interesting.

**Review by dnault — CHANGES_REQUESTED** [review]
Nice improvement!

File: modules/fts/pages/fts-supported-queries-boolean-query.adoc (hunk 1 above)
Comment (abhinavdangeti) [reply modules/fts/pages/fts-supported-queries-boolean-query.adoc]: Shoot thanks.

Ya should look into updating the min part separately.

**Review by dnault — APPROVED** [review]

**Comment by RayOffiah** [comment]
@abhinavdangeti <-- Could you fix the merge conflicts on this PR? Thanks

I'll arrange for you to be added to the set-reviewer list.

**Comment by abhinavdangeti** [comment]
@RayOffiah Done.

**Review by RayOffiah — APPROVED** [review]

**Pull request merged**